CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2020-10-16
+
+### Added
+
+ - Codelist checks
+ - Additional checks for; currency specified, empty values, project prefixes and the presence of org id references.
```

Comment:
I don't think this needs to block the PR or release, but I think this should be ":" rather than ";".